docs/run/advanced-runtime-options.mdx

```diff
@@ -52,13 +52,13 @@ options.execution.shots = 2048
 ```
 ## Pass options to a primitive
 
-### Options class 
+### Options class
 
-When creating an instance of the `Estimator` or `Sampler` class, you can pass in the `options` you created in the options class. Those options will then be applied when you use `run()` to perform the calculation.  Example: 
+When creating an instance of the `Estimator` or `Sampler` class, you can pass in the `options` you created in the options class. Those options will then be applied when you use `run()` to perform the calculation.  Example:
 
 <Tabs>
   <TabItem value="PrimV2" label="V2 primitives">
-    `SamplerV2` and `EstimatorV2` have separate options classes that do not need to be instantiated.  You can see the available options and update option values during or after primitive initialization.  Those options will then be applied when you use `run()` to perform the calculation.  Example: 
+    `SamplerV2` and `EstimatorV2` have separate options classes that do not need to be instantiated.  You can see the available options and update option values during or after primitive initialization.  Those options will then be applied when you use `run()` to perform the calculation.  Example:
```

Comment:
This feels a bit redundant from line 57 (also applies to the analogous place for V1 in line 74).

Reply:
I can see that it feels awkward, but I think I'll leave it because of the V1 tab, and also because it's a change from V1, so we want to make sure people understand.